### PR TITLE
Trad/management

### PR DESCRIPTION
**Pull request by fdechelle**
translated management module:

- renamed gestion/ to management/
- translated all files

### CODE REVIEW

File: source/modules/management/contract.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+   :alt: Main fields of a contract
+   :align: center
+
+.. include:: ../tabs/templates.rst
+
+.. note::
+
+   It is possible to receive notifications in order to be informed of events such as contract end.
+   When contract must be renewed expressly, it is useful to be notified before end of contract. For periodic contracts, it is also possible to be notified at the end of each period.
+   Notifications are configured at entity level, in menu **Configuration > Notifications** to define models and destination and in menu **Configuration > Entity** to enable notification, define default values and a possible anticipation of notification.
```
Comment (cconard96) [inline source/modules/management/contract.rst:22]:
```suggestion
   Notifications are configured at entity level, in menu **Setup > Notifications** to define models and destination and in menu **Administration > Entities** to enable notification, define default values and a possible anticipation of notification.
```

File: source/modules/management/documents.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+The file corresponding to the document can be added in several ways:
+
+* From local disk;
+* Using a web link pointing to a document (image, HTML page, PDF...);
+* Using a file previously transferred with FTP to sub-folder */files/_uploads/* of GLPI installation
+
+A MIME type can also be given.
+
+.. note::
+   * Document types authorized in GLPI according to their extension are defined in **Configuration** > **Drop-downs** > **Management** > **Document types**
```
Comment (cconard96) [inline source/modules/management/documents.rst:18]:
```suggestion
   * Document types authorized in GLPI according to their extension are defined in **Setup** > **Dropdowns** > **Management** > **Document types**
```

File: source/modules/management/licenses.rst [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+* Inventorying licenses, with links to softwares inventoried in GLPI;
+* Following license installations on all assets;
+* Linking licenses to financial management;
+* Anticipating and following licenses renewal.
+
+.. note:: 
+        
+   A license cannot exists in GLPI without a software associated to the license when creating it.
+
+   License management is not automated, a human following is needed for informations update.
```
Comment (cconard96) [inline source/modules/management/licenses.rst:15]:
```suggestion
   License management is not automated, a human follow-up is needed for information update.
```
Or
License management is not automated, a manual follow-up is needed for information update.

**Review by cconard96 — CHANGES_REQUESTED** [review]

**Review by trasher — APPROVED** [review]
OK for me, see Curtis comments :)

**Review by cconard96 — APPROVED** [review]

**Pull request merged**